### PR TITLE
Create default public key endpoint for radar-gateway

### PR DESCRIPTION
**Pull request by pvannierop**
# Problem
Previous PR introduced public key endpoint for radar-gateway. This implementation did not include config for a sensible default value.

# Solution
This PR will provide a default value that respects the protocol (http, https) which is needed for local testing.

### CODE REVIEW

File: charts/radar-gateway/templates/configmap.yaml [hunk 1]
```diff
@@ -57,6 +57,7 @@ data:
       managementPortalUrl: {{ .Values.managementportal_url }}
       checkSourceId: {{ .Values.checkSourceId }}
       publicKeyUrls:
+        - {{ printf "%s://%s/managementportal/oauth/token_key" .Values.advertised_protocol .Values.serverName | quote }}
```
[the remark this answers is not in the document]
Comment (pvannierop) [reply charts/radar-gateway/templates/configmap.yaml:60]: @keyvaann I think so yes.

[the remark this answers is not in the document]
Comment (pvannierop) [reply charts/radar-gateway/templates/configmap.yaml:60]: @keyvaann Ah I think that I understand where your question comes from. This is related to the error that is thrown by gateway when using this config option. Shall I enable this via config option and have it off by default?

**Review by keyvaann — APPROVED** [review]
LGTM

**Comment by keyvaann** [comment]
@pvannierop The lint is failing.

File: charts/radar-gateway/templates/configmap.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (pvannierop) [reply charts/radar-gateway/templates/configmap.yaml:61]: @keyvaann  I have added the option to disable the publicKeyUrls config option.

[the remark this answers is not in the document]
Comment (keyvaann) [reply charts/radar-gateway/templates/configmap.yaml:61]: Thanks!

**Pull request merged**